docs/developer/client.md

```diff
@@ -1,93 +1,85 @@
 When running a Software Delivery Machine (SDM) or lower-level Atomist
 API client, you interact with the Atomist development automation API
-via a client [WebSocket][ws] connection.  WebSocket connections are
+via a [WebSocket][ws] connection.  WebSocket connections are
 persistent, providing bidirectional communication between the client
-and the API.  As such, the client process is a persistent process with
+and the API. The SDM subscribes to events published by the API.
+ As such, the SDM is a persistent process with
 a lifecycle that is more like a traditional _server_ process.  This
-section documents creating, building, and running an automation
-client, discusses each part of the client lifecycle, and details the
-structure and organization of a typical automation client project.
+section documents creating, building, and running an SDM, 
+discusses each part of the SDM lifecycle, and details the
+structure and organization of a typical SDM project.
 
-Before you can build and run your own automation client,
+Before you can build and run your own SDM,
 {!prereq-items.md!}
 
 [ws]: https://en.wikipedia.org/wiki/WebSocket (WebSocket)
 
-## Creating a client project
+## Creating an SDM project
 
 An Atomist API client project is any project that connects to the
-Atomist development automation API.  The reference implementation of
-the automation client is the [automation-client-ts][client-ts]
-library, which is written in [TypeScript][ts].  Client projects, like
-any SDM, depends on the automation-client library and uses its
+Atomist development automation API.  The reference implementation is the [automation-client-ts][client-ts]
+library, which is written in [TypeScript][ts]. Any SDM
+ depends on the @atomist/automation-client library and uses its
 interface to connect to the Atomist API.  The combination of
-TypeScript and [GraphQL][gql] provides an excellent development
-experience, with excellent tooling and debugging support.
+TypeScript and [GraphQL][gql] provides a development
+experience with excellent tooling and debugging support.
 
-There are a few ways to create a new automation client project.  We
-suggest using the [automation-seed-ts][seed] project as a seed for
-your automation client project.
+There are a few ways to create a new SDM project.  We
+suggest using the [blank-sdm-seed][seed] project as a seed for
+your automation client project. You can do this localluy with the Atomist CLI: `atomist create sdm`.
```

Comment:
spelling: `localluy`

Reply:
Also best to break the command line into a code block so people can easily cut and paste.

```
atomist create sdm
```